packages/format-library/src/text-color/inline.native.js

```diff
@@ -148,6 +148,7 @@ export default function InlineColorUI( { name, value, onChange, onClose } ) {
 			contentStyle={ { paddingLeft: 0, paddingRight: 0 } }
 			hasNavigation
 			leftButton={ null }
+			testID="inline-text-color"
```

Comment:
Nitpick: On other test ID definitions related to bottom sheets, we used the suffix `-modal` ([example](https://github.com/WordPress/gutenberg/blob/54049045c46380d0d05f54761fd792fe6faef4f9/packages/editor/src/components/editor-help/index.native.js#L87)) to identify that it's associated to a modal.

Reply:
Good idea, thanks! I'll update it 👍